{{ cookiecutter.repo_name }}/deploy/docker-compose.yml

```diff
@@ -1,68 +1,27 @@
 version: '2'
 services:
-    worker:
-        image: drtools/{{cookiecutter.repo_name}}-worker:dev
-        build:
-          context: .
-          dockerfile: dev.dockerfile
-
     luigid:
-        image: drtools/{{cookiecutter.repo_name}}-worker:dev
+        image: axiom/docker-luigi
         container_name: luigid
-        build:
-          context: .
-          dockerfile: dev.dockerfile
         ports:
             - "8082:8082"
 
         command: luigid
 
     dask-scheduler:
-      image: drtools/{{cookiecutter.repo_name}}-worker:dev
-      build:
-        context: .
-        dockerfile: dev.dockerfile
-
+      image: drtools/drtools:latest
       command: dask-scheduler
       container_name: dask-scheduler
 
       depends_on:
           - dask-worker
-
-      environment:
-          - PATH=/home/ubuntu/.local/bin:$PATH
-          - LC_ALL=C.UTF-8
-          - LANG=C.UTF8
-          - LOCAL_GROUP_ID
-          - LOCAL_USER_ID
-          - LOCAL_USER
-
       ports:
         - "8787:8787"
         - "8788:8788"
 
 
     dask-worker:
-      image: drtools/{{cookiecutter.repo_name}}-worker:dev
```

Comment:
But if dask-worker doesn't have access to project code, how can it perform computations?

Reply:
I think there are very little situations where the dask-worker needs to have project specific code. Project specific code should be transferred to dask with cloudpickle :)

Reply:
I remember having some issues with it... Probably when dask worker had access to another branch of the project then the main worker. Not sure about it though.